nipype/interfaces/ants/registration.py

```diff
@@ -49,8 +49,14 @@ class ANTSInputSpec(ANTSCommandInputSpec):
     metric = traits.List(traits.Enum('CC', 'MI', 'SMI', 'PR', 'SSD',
                                      'MSQ', 'PSE'), mandatory=True, desc='')
 
-    metric_weight = traits.List(traits.Float(), requires=['metric'], desc='')
-    radius = traits.List(traits.Int(), requires=['metric'], desc='')
+    metric_weight = traits.List(traits.Float(), value=[1.0], usedefault=True,
+                                requires=['metric'], mandatory=True,
+                                desc='the metric weight(s) for each stage. '
+                                'The weights must sum to 1 per stage.')
```

Comment:
isn't this a pep8 warning?

Reply:
this line doesn't give any warning, but the files indeed violate many. will edit

Reply:
I will actually review pep8 only for the first part I was changing, will review pep8 for entire file together with other updates